test/cri-containerd/main.go

```diff
@@ -237,6 +262,13 @@ func pullRequiredImagesWithLabels(t *testing.T, images []string, labels map[stri
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	var auth *runtime.AuthConfig
+	if token != "" {
+		auth = &runtime.AuthConfig{
+			Username: cpugroup.NullGroupID,
```

Comment:
This doesn't seem right to take a dependency on `cpugroup` package here, since it's purpose is unrelated. What are we trying to pass here?

Reply:
when using an access token username is set to the zero `UUID` ([reference](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication#az-acr-login-with---expose-token)), which I believe the `NullGroupID` is essentially, so I just wanted to reuse the const. after digging a bit more, I'll probably change this to be an actual `username` ([reference](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-repository-scoped-permissions#authenticate-with-token))

Reply:
removed it and hardcoded a default